CHANGELOG.md

```diff
@@ -0,0 +1,30 @@
+# Release Notes for autonomX
+
+## Version 0.1.0 - 2020-09-09
+
+This is the first release of autonomX.
+
+New features:
+
+- Run a SNN generator
+- Edit the properties of the SNN generator
+- View the current value of the generator output node
+- View the evolution of the output node of the generator over time
+- See a 2D lattice representation
+- Send the output node of my SNN generator through an OSC output
+- Configure the parameters of the OSC output
+- Receive OSC to the input node of my SNN generator
+- Configure the parameters of the OSC input
+- Have multiple inputs and outputs
+- Draw the inputs and outputs on the 2D lattice representation
+- See different colors for each inputs and outputs
+- Receive OSC from the outside into the inputs of the generator
+- Send the outputs of the generator to the outside with OSC
```

Comment:
These entries are duplicates of "Send the output node of my SNN generator through an OSC output" and "Send the output node of my SNN generator through an OSC output" although their wording seems a bit better.